challenges/repeatedWord/repeated-word.js

```diff
@@ -0,0 +1,14 @@
+module.exports = function repeatedWord(string) {
+  let obj = {};
+  let stringWord = string.split(' ');
+  for(let i = 0; i < stringWord.length; i++) {
+    if(!obj[stringWord[i]]) obj[stringWord[i]] = 1;
+    else obj[stringWord[i]]++;
```

Comment:
instead of doing the new array/Obj.keys thing, if something is repeated, it will exist in the object. Your "else" could just be to return obj[stringWord[i]]